packages/flutter_tools/test/dependency_checker_test.dart

```diff
@@ -67,5 +70,17 @@ void main()  {
       // the .dart file.
       expect(dependencyChecker.check(baseTime), isTrue);
     });
+
+    /// Test a flutter tool move.
+    ///
+    /// Tests that the flutter tool doesn't crash and displays a warning when its own location
+    /// changed since it was last referenced to in a package's .packages file.
+    testUsingContext('moved flutter sdk', () async {
+      fs.currentDirectory = path.join(dataPath, 'changed_sdk_location');
```

Comment:
This will affect the CWD for all isolates in the current process - it's what's causing Travis to complain.